### PR TITLE
Styling fix for components

### PR DESCRIPTION
**Pull request by toha993**

### CODE REVIEW

File: packages/react/src/input-base/InputBase.css.ts [hunk 1]
```diff
@@ -82,11 +78,6 @@ export const input = recipe({
         },
       },
     }),
-    responsiveStyle({
```
Comment (radnan) [inline packages/react/src/input-base/InputBase.css.ts:85]: let's revert this change please - we can add a comment here on why this was done - but the idea here is to explicitly use `16px` on mobile because otherwise iOS will zoom in on the page anytime the input is focused

https://defensivecss.dev/tip/input-zoom-safari/

File: packages/react/src/input-base/InputBase.css.ts [hunk 2]
```diff
@@ -63,10 +58,11 @@ export const input = recipe({
     {
       bg: "transparent",
       flex: "auto",
+      fontWeight: "400",
     },
     style({
-      fontSize: "16px",
-      lineHeight: "22px",
```
Comment (radnan) [inline packages/react/src/input-base/InputBase.css.ts:69]: we need to revert this change - and we can add a comment explaining it - line-height was set to `22px` to adjust for the `1px` border on top and bottom

File: packages/react/src/input-base/InputBase.css.ts (hunk 1 above)
Comment (toha993) [reply packages/react/src/input-base/InputBase.css.ts:85]: Got it vai.

**Comment by radnan** [comment]
this PR seems to have a lot of different fixes all rolled together - can we please split them up at least by component? otherwise it's a bit hard to review the changes
